docs/use-github-in-bcgov/bc-government-organizations-in-github.md

```diff
@@ -41,31 +40,109 @@ The `bcgov-c` organization stores temporary (up to 12 months), private repositor
+  
+To keep access to the bcgov GitHub organization, all users must link their IDIRs to their GitHub IDs between **June 30 and July 30, 2024**. If users don’t complete this by July  30, 2024, they’ll be automatically removed from the bcgov GitHub organization, and by extension, all the code repositories connected to bcgov.   
+Contractors without IDIRs must get one to before they can enable SSO. [Learn how to order an IDIR](). 
+ 
+[Learn more about the timelines and instructions](bc-govermentorganizations-in-github.md#the-changes). 
+ 
+### User impacts 
+ 
+Users who have accounts attached to the [bcgov GitHub organization](https://github.com/bcgov). 
+ 
+Users who use GitHub as their authentication method to login to: 
+
+- [Private Cloud](https://developer.gov.bc.ca/docs/default/component/platform-developer-docs),  
+- [Public Cloud](https://developer.gov.bc.ca/docs/default/component/public-cloud-techdocs), 
+- [Stack Overflow](https://stackoverflow.developer.gov.bc.ca/),  
+- [Rocket.chat](https://chat.developer.gov.bc.ca/),  
+- Any other custom apps owned by the province that use membership in the bcgov organization to authenticate and authorize users 
+ 
+Teams relying on integrations that need access to the bcgov org, for example: 
+
+- GitHub or OAuth apps 
+  - to automate the release or testing of your code, including 3rd party security tools 
+- Access tokens and/or SSH keys that your app or team may rely on that will be disconnected when user access is removed 
+  - IDE or command line tools and/or automation scripts 
+ 
+### The changes 
+ 
+Before this upgrade, users only needed their personal GitHub IDs to access the bcgov GitHub organization as a member. 
+ 
+By July 30, 2024, users must follow prompts in GitHub to link their personal GitHub ID with their IDIR account.  If not, users will be automatically removed. 
+ 
+[Learn more about how linking works]().  
+ 
+Ministries won’t be charged for upgraded GitHub licenses.  However, contractors need IDIRs before they can enable SSO. Ministries may need to seek expense approvals to order [IDIRs for contracted resources without one](). 
+  
+### Directions to link your accounts before July 30
+ 
+#### For employees 
+ 
+[placeholder for video to show what to expect]()
+ 
+1. Have any two-factor devices, and personal GitHub login and IDIR login ready for linking 
+2. Log into GitHub and follow the prompts on the pop up to connect your GitHub and IDIR accounts together. [Learn more about how this works](). 
+3. Reauthorize your personal access tokens and/or SSH keys so they continue to work with SSO enabled. [Learn more about how to do this](). 
+ 
+That’s it, your accounts are now linked.  
+ 
+#### For contractors 
+ 
+If you don’t have an IDIR account, [we have guidance on ordering a compatible IDIR] ](faq link). It can take several days to complete depending on your ministry.We recommend doing with this step as soon as possible to avoid access disruptions.  
```

Comment:
[we have guidance on ordering a compatible IDIR] ](https://bcdevex.atlassian.net/wiki/spaces/DEVEXP/pages/1514274870/GitHub+Single+Sign-On+SSO+Transition+Guide)